source/plugin/project/gradle.rst

```diff
@@ -114,18 +120,17 @@ Without SpongeGradle
 Generally, everything necessary to compile a Sponge plugin using Gradle can be done by simply adding the SpongeAPI
 dependency to your project:
```

Comment:
This is not as true anymore -- you'd have to manually write out a metadata file

Reply:
Or use https://github.com/SpongePowered/SpongeDocs/blob/stable/source/plugin/plugin-meta.rst#the-sponge_pluginsjson-format